tests/testthat/error/test-bind.txt

```diff
@@ -5,7 +5,7 @@ vec_rbind() fails with complex foreign S3 classes
 > x <- structure(foobar(1), attr_foo = "foo")
 > y <- structure(foobar(2), attr_bar = "bar")
 > vec_rbind(set_names(x, "x"), set_names(y, "x"))
-Error: Can't combine `..1$x` <vctrs_foobar> and `..2$x` <vctrs_foobar>.
+Error: Can't combine `..1` <vctrs_foobar> and `..2` <vctrs_foobar>.
```

Comment:
I think you could now remove the `set_names()` in the previous line since it's no longer used